README.md

```diff
@@ -111,6 +111,12 @@ curl --include \
 4. localhost:8006/paid_invoice_from_xendit (callback url) get the paid invoice data.
 ```
 
+# Issue credit card refund
+```
+cd src/
+php examples/issue_credit_card_refund.php [amount] [external_id] [optional-idempotency-key]
```

Comment:
Adding example means adding additional 2 lines like this https://github.com/xendit/xendit-php-clients/pull/17/files#diff-04c6e90faac2675aa89e2176d2eec7d8R31